app/src/main/java/org/wikipedia/descriptions/DescriptionEditFragment.kt

```diff
@@ -321,6 +321,7 @@ class DescriptionEditFragment : Fragment() {
                                     AnonymousNotificationHelper.onEditSubmitted()
                                     waitForUpdatedRevision(newRevId)
                                     EditAttemptStepEvent.logSaveSuccess(pageTitle, EditAttemptStepEvent.INTERFACE_OTHER)
+                                    MachineGeneratedArticleDescriptionsAnalyticsHelper.logActualPublishedDescription(requireContext(), binding.fragmentDescriptionEditView.description.orEmpty())
```

Comment:
It looks like these events will be sent in all cases of description editing, not just adding machine-generated descriptions. Is that intentional?